codefresh/apk.yml

```
@@ -95,9 +99,10 @@ steps:
           - APK_PACKAGES_PATH=${{CF_VOLUME_PATH}}/artifacts/3.9
         commands:
           - make -C vendor build
+          - make -C vendor index
           - find "${APK_PACKAGES_PATH}" -type f -name '*.apk' | xargs ls -l 
```

This will produce a column-aligned list instead of a ragged one
```suggestion
           - find "${APK_PACKAGES_PATH}" -type f -name '*.apk' -exec ls -l {} +
```